src/Files.Backend/Services/IFileExplorerService.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using Files.Sdk.Storage.LocatableStorage;
+
+namespace Files.Backend.Services
+{
+    /// <summary>
+    /// A service that interacts with the system file explorer.
+    /// </summary>
+    public interface IFileExplorerService
+    {
+        /// <summary>
+        /// Opens the app folder.
+        /// </summary>
+        /// <param name="cancellationToken">A <see cref="CancellationToken"/> that cancels this action.</param>
+        /// <returns>A <see cref="Task"/> that represents the asynchronous operation.</returns>
+        Task OpenAppFolderAsync(CancellationToken cancellationToken = default);
```

Comment:
Very good to see proper comments, nicely done!

Reply:
Copied them from SFFS 😅 
We should definitely comment our code as part of the refactor (and also include <inheritdoc/>)

Reply:
Can you add this to the code guidelines?